docs/source/tracking.rst

```diff
@@ -411,13 +411,15 @@ default location to store artifacts for all runs in this experiment. Additional,
 is a property on :py:class:`mlflow.entities.RunInfo` to indicate location where all artifacts for
 this run are stored.
 
-Use ``--default-artifact-root`` (defaults to local ``./mlruns`` directory) to configure default
-location to server's artifact store. This will be used as artifact location for newly-created
+Use ``--default-artifact-root`` (defaults to client's local ``./mlruns`` directory) to configure the default
```

Comment:
IMHO artifact paths are always interpreted from the client. E.g. I observe that, when I run `mlflow server` on a remote machine in e.g. `/mnt/bla` that my client tries to write artifacts to its local `/mnt/bla/mlruns/` which it probably can't because that path doesn't exist. We can rephrase to
```
defaults to server's local ``./mlruns`` path which will be used from the client
``` 
?